client/state/action-types.js

```diff
@@ -983,6 +983,7 @@ export const THEME_UPLOAD_FAILURE = 'THEME_UPLOAD_FAILURE';
 export const THEME_UPLOAD_PROGRESS = 'THEME_UPLOAD_PROGRESS';
 export const THEME_UPLOAD_START = 'THEME_UPLOAD_START';
 export const THEME_UPLOAD_SUCCESS = 'THEME_UPLOAD_SUCCESS';
+export const THEMES_BANNER_IS_SHOWING = 'THEMES_BANNER_IS_SHOWING';
```

Comment:
An action type should bear a more imperative (or at least verb-like, rather than state-describing) name, so it should probably be `THEMES_BANNER_HIDE`.

Reply:
(or `THEMES_BANNER_VISIBILITY_TOGGLE`, if you want to make it more generic.)

Reply:
I think `_HIDE` is better tho since it's easier to grok, and we can always add `_SHOW`.

Reply:
Good idea, done! 👍